src/kOS.Safe/Encapsulation/StringValue.cs

```diff
@@ -213,34 +213,17 @@ private void StringInitializeSuffixes()
             return !(val1 == val2);
         }
 
-        public static bool operator ==(StringValue val1, string val2)
```

Comment:
I needed to remove these because of the implicit conversion that means the compiler cant figure out if it should do the `<string, string>` convert or the `<StringValue,string>`